Objective-C/CBLReplicator.mm

```diff
@@ -454,6 +455,38 @@ - (void) removeChangeListenerWithToken: (id<CBLListenerToken>)token {
     }
 }
 
+- (NSSet<NSString*>*) pendingDocumentIds: (NSError**)error {
+    if (_config.replicatorType > 1) {
+        if (error)
+            *error = [NSError errorWithDomain: CBLErrorDomain
+                                         code: CBLErrorUnsupported
+                                     userInfo: @{NSLocalizedDescriptionKey: kCBLErrorMessagePullOnlyPendingDocIDs}];
+        return [NSSet set];
+    }
+    
+    if (!_repl) {
+        CBLLogInfo(Sync, @"Trying to fetch pending documentIds without a c4replicator %@", _repl);
+        return [NSSet set];
+    }
+    
+    C4Error c4err = {};
+    C4SliceResult result = c4repl_getPendingDocIDs(_repl, &c4err);
+    if (c4err.code > 0) {
+        convertError(c4err, error);
+        CBLWarnError(Sync, @"Error while fetching pending documentIds: %d/%d", c4err.domain, c4err.code);
+        return [NSSet set];
+    }
+    
+    if (!result.size)
```

Comment:
I feel easier to understand when doing result.size > 0.

Reply:
```suggestion
    if (result.size <= 0)
```